docs/content/en/docs/concepts/support-scope.md

```diff
@@ -2,39 +2,83 @@
-For official Ubuntu support, see the Canonical [Support](https://ubuntu.com/support) page.
+- [EKS Anywhere Pricing Page](https://aws.amazon.com/eks/eks-anywhere/pricing/)
+- [EKS Anywhere FAQ Page](https://aws.amazon.com/eks/eks-anywhere/faqs/)
+- [Steps to purchase a subscription]({{< relref "../clustermgmt/support/purchase-subscription" >}})
+- [Steps to license your cluster]({{< relref "../clustermgmt/support/cluster-license" >}})
+- [Steps to share curated packages with another account]({{< relref "../clustermgmt/support/share-packages" >}})
 
-* **Red Hat Enterprise Linux (RHEL)**: EKS Anywhere has been tested with RHEL-based nodes.
-As with Ubuntu, Amazon will assist with troubleshooting and configuration guidance with RHEL-based nodes under the [EKS Anywhere Enterprise Subscription.](https://aws.amazon.com/eks/eks-anywhere/pricing/)
-To build your own RHEL-based EKS Anywhere node image, refer to [Building node images]({{< relref "../osmgmt/artifacts/#building-node-images" >}}).
-For official Red Hat support, see the [Red Hat Enterprise Linux Subscriptions](https://www.redhat.com/en/store/linux-platforms) page.
+If you are using EKS Anywhere and have not purchased a subscription, you can file an [issue](https://github.com/aws/eks-anywhere/issues) in the EKS Anywhere GitHub Repository, and someone will get back to you as soon as possible. If you discover a potential security issue in this project, we ask that you notify AWS/Amazon Security via the [vulnerability reporting page.](http://aws..com/security/vulnerability-reporting/) Please do not create a public GitHub issue for security problems.
 
-## Curated packages support
-Amazon [EKS Anywhere Curated Packages]({{< relref "../packages/" >}}) are Amazon-curated software packages that extend the core functionalities of Kubernetes on your EKS Anywhere clusters.
-All curated packages, including the curated OSS packages, are supported under the EKS Anywhere Enterprise Subscription.
+### FAQs
+
+**1. How much does an EKS Anywhere Enterprise Subscription cost?**
+
+For pricing information, visit the [EKS Anywhere pricing page](https://aws.amazon.com/eks/eks-anywhere/pricing/).
+
+**2. How can I purchase an EKS Anywhere Enterprise Subscription?**
+
+Go to the [EKS Anywhere Enterprise Subscription section](https://console.aws.amazon.com/eks/home#/eks-anywhere) of the EKS console. In the console, select your preferred term (1-year or 3-year). Then, specify the license quantity, configure tags and automatic renewal. Review the pending purchase. When the purchase is submitted, you can review licenses associated with your subscription in the console, accept the license grants with AWS License Manager, apply the licenses to your EKS Anywhere clusters, and access EKS Anywhere Curated Packages. For more information, reference the [EKS Anywhere documentation.]({{< relref "../clustermgmt/support/purchase-subscription" >}})
```

Comment:
```suggestion
Go to the [EKS Anywhere Enterprise Subscription section](https://console.aws.amazon.com/eks/home#/eks-anywhere) of the EKS console. In the console, select your preferred term (1-year or 3-year). Then, specify the license quantity, configure tags, and choose automatic renewal (optional). Review the pending purchase. When the purchase is submitted, you can review licenses associated with your subscription in the console, accept the license grants with AWS License Manager, apply the licenses to your EKS Anywhere clusters, and access EKS Anywhere Curated Packages. For more information, reference the [EKS Anywhere documentation.]({{< relref "../clustermgmt/support/purchase-subscription" >}})
```
This is a pretty specific list for an FAQ section. If this information is elsewhere, you might want to point there and not risk this text going out of date.

Reply:
ack, linked out to the docs instead